lib/eventsource.js

```diff
@@ -232,16 +235,31 @@ function EventSource (url, eventSourceInitDict) {
       var buf
       var startingPos = 0
       var startingFieldLength = -1
+      let sizeUsed = 0
```

Comment:
We should replace `var` with `let` also for the above?

Reply:
Realistically I would like to change almost everything about this, but for this PR I tried to limit the changes to things I was directly modifying.